source/includes/administration/_invoices.md

```diff
@@ -0,0 +1,149 @@
+
+`GET /invoices/find/:reseller_id/customer_invoices`
+
+Retrieve a list of invoices belonging to the customers of the reseller specified.
+
+```shell
+# Retrieve customer invoices list
+curl "https://cloudmc_endpoint/rest/invoices/find/efd32752-c6f2-45cf-b494-cc6be8a45845/customer_invoices?includeAllSubOrgs=false&billingCycle=07-2021" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+	"data": [{
+		"draftedDate": "2021-07-27T00:00:00Z",
+		"createdDate": "2021-07-27T00:00:00Z",
+		"status": "DRAFT",
+		"organization": {
+			"name": "org_name",
+			"id": "289ec5fb-0970-44e3-bca8-777a691e23c7"
+		},
+		"id": "20e9b8d8-b1cb-4462-b6e8-fbb8416b2cbb",
+		"detail": {
+			"discounted": true,
+			"discounts": [],
+			"startDate": "2021-07-01T00:00:00Z",
+			"endDate": "2021-08-01T00:00:00Z",
+			"currency": "CAD",
+			"subTotal": 272154.29,
+			"categories": [{
+				"discounted": true,
+				"unit": {
+					"unit": "HOUR",
+					"name": {}
+				},
+				"discounts": [],
+				"productId": "5bec5db2-b438-4610-ba73-f3ed50a7b26b",
+				"price": "40.0",
+				"name": {
+					"en": "memory",
+					"fr": "memory"
+				},
+				"subTotal": 65407.39,
+				"sku": "product_sku"
+			}]
+		}
+	}]
+}
+```
+
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`includeAllSubOrgs`<br/>*boolean* | Whether or not to include sub-organizations of sub-organizations or just direct sub-organizations of the reseller.
+`billingCycle`<br/>*String* | The cycle which the invoice belongs to. Format is `MM/YYYY`
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the invoice.
+`status`<br/>*enum* | The status of the invoice. Possible values are: USAGE_PENDING, DRAFT, ISSUED, OVERDUE, PAID, VOID.
+`createdDate`<br/>*date* | The created date of the invoice.
+`draftedDate`<br/>*date* | The date the invoice was drafted. Is null if status is still USAGE_PENDING.
+`issuedDate`<br/>*date* | The date the invoice was issued.
+`dueDate`<br/>*date* | The date the invoice is due.
+`organization.id`<br/>*UUID* | The UUID of the organization the invoice belongs to.
+`organization.name`<br/>*string* | The name of the organization the invoice belongs to.
+`detail`<br/>*Object* | The invoice detail contains a currency and a list of categories. Each category has a list of products.
+`total`<br/>*BigDecimal* | The invoice total.
+`balance`<br/>*BigDecimal* | The invoice's remaining balance.
+
+<!-------------------- LIST INVOICES -------------------->
+### List invoices of organization
+
+`GET /invoices?organization_id=:organization_id&billingCycle=MM-YYYY`
+
+Retrieve the list of invoices associated with an organization.
+
+```shell
+# Retrieve applied pricing list
+curl "https://cloudmc_endpoint/rest/invoices?organization_id=289ec5fb-0970-44e3-bca8-777a691e23c7&billingCycle=07-2021" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
```

Comment:
should I only be showing this for one of the endpoints since it's the same for both? same for attributes

Reply:
It's ok to duplicate.